### PR TITLE
feat(integration-templates): add discourse integration template

### PR DESCRIPTION
**Pull request by hassan254-prog**
## Describe your changes

- Add discourse integration template

## Checklist before requesting a review (skip if just adding/editing APIs & templates)
- [ ] I added tests, otherwise the reason is: 
- [ ] I added observability, otherwise the reason is:
- [ ] I added analytics, otherwise the reason is:

### CODE REVIEW

File: integration-templates/discourse/nango.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+integrations:
+    discourse:
+        syncs:
+            users:
+                description: |
+                    Fetches a list of active users from Discourse.
+                endpoint: /discourse/users
+                sync_type: full
+                runs: every 1 hour
```
Comment (khaliqgant) [inline integration-templates/discourse/nango.yaml:9]: track_deletes: true

Comment (hassan254-prog) [reply integration-templates/discourse/nango.yaml:9]: Thank you Khaliq. Updated with [commit](https://github.com/NangoHQ/nango/pull/2722/commits/4809e695a54651800f1f90a89dbf86d0880b5335)

File: integration-templates/discourse/nango.yaml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+integrations:
+    discourse:
+        syncs:
+            users:
+                description: |
+                    Fetches a list of active users from Discourse.
```
Comment (bodinsamuel) [inline integration-templates/discourse/nango.yaml:6]: I don't think we filter
```suggestion
                    Fetches a list of users from Discourse.
```

**Review by bodinsamuel — APPROVED** [review]

File: integration-templates/discourse/nango.yaml (hunk 2 above)
Comment (hassan254-prog) [reply integration-templates/discourse/nango.yaml:6]: The implemented endpoint is filtering for active users only  `/active`.

Comment (bodinsamuel) [reply integration-templates/discourse/nango.yaml:6]: ah yeah I didn't saw it was in the path of the endpoint my bad. 
Is it still relevant to have `active` in the User model then? or maybe the sync should be renamed to users-active. Not suggesting to change anything but wondering if we want to be explicit

Comment (hassan254-prog) [reply integration-templates/discourse/nango.yaml:6]: I think that would be nice to explicitly name the script as `active-users` as it only fetches for active users only.

**Review by bodinsamuel — APPROVED** [review]

**Pull request merged**